### ISSUE TITLE
relation breaks

### ISSUE THREAD

**Issue by azamaulanaaa** [drizzle-team/drizzle-graphql#10]
package

```
    "drizzle-graphql": "^0.7.0",
    "drizzle-orm": "^0.30.10",
```

query

```graphql
query ($entity_id: Int!, $date: String!) {
  entitiesSingle(where: { id: { eq: $entity_id } }) {
    financeSalaries(where: { activatedAt: { lte: $date } }) {
      value
      activatedAt
    }
  }
}
```

response

```json
{
  "errors": [
    {
      "message": "Cannot return null for non-nullable field EntitiesSelectItem.financeSalaries.",
      "locations": [
        {
          "line": 3,
          "column": 5
        }
      ],
      "path": [
        "entitiesSingle",
        "financeSalaries"
      ]
    }
  ],
  "data": {
    "entitiesSingle": null
  }
}
```

Note:

in version 0.6, everything works just fine. Updating to 0.7 got the problem. thanks

**Comment by Sukairo-02** [comment]
What's your drizzle schema?

**Comment by azamaulanaaa** [comment]
this is my schema hope it helps


```typescript
export const entities = pgTable("entities", {
  id: serial("id").primaryKey().notNull(),
  createdAt: timestamp("created_at").defaultNow().notNull(),
  name: text("name").notNull(),
});

export const entitiesRelations = relations(entities, ({ many }) => ({
  financeSalaries: many(financeSalaries),
}));


export const financeSalaries = pgTable(
  "finance_salaries",
  {
    id: serial("id").primaryKey().notNull(),
    createdAt: timestamp("created_at").defaultNow().notNull(),
    activatedAt: timestamp("activated_at").defaultNow().notNull(),
    value: jsonb("value").notNull(),
    entityId: integer("entity_id")
      .notNull()
      .references(() => entities.id, {
        onDelete: "restrict",
        onUpdate: "cascade",
      }),
  },
  (table) => ({
    activatedAtIndex: index().on(table.activatedAt),
    entityIdIndex: index().on(table.entityId),
    entityIdValueUnique: unique().on(table.entityId, table.value),
  }),
);

export const entitySalariesRelations = relations(
  financeSalaries,
  ({ one }) => ({
    entity: one(entities, {
      fields: [financeSalaries.entityId],
      references: [entities.id],
    }),
  }),
);
```

**Comment by Edsol** [comment]
same problem but on mysql database, so it does not depend on dbms

**Comment by Sukairo-02** [comment]
Fixed - [v0.7.2](https://github.com/drizzle-team/drizzle-graphql/releases/tag/0.7.2)
